specs/lib/gltfPrimitiveToCesiumGeometrySpec.js

```diff
@@ -11,20 +11,24 @@ var positions = new Float64Array([ -0.5, -0.5, 0.5, 0.5, -0.5, 0.5, -0.5, 0.5, 0
 var normals = new Float32Array([ 0, 0, 1, 0, 0, 1, 0, 0, 1, 0, 0, 1, 1, 0, 0, 1, 0, 0, 1, 0, 0, 1, 0, 0, 0, 1, 0, 0, 1, 
     0, 0, 1, 0, 0, 1, 0, 0, -1, 0, 0, -1, 0, 0, -1, 0, 0, -1, 0, -1, 0, 0, -1, 0, 0, -1, 0, 0, -1, 0, 0, 0, 0, -1, 0, 0,
     -1, 0, 0, -1, 0, 0, -1 ]);
+var sts = new Float32Array([ 6, 0, 5, 0, 6, 0.9999998807907104, 5, 0.9999998807907104, 4, 0, 5, 0, 4, 1, 5, 1,
+    2, 0, 1, 0, 2, 1, 1, 1, 3, 0, 4, 0, 3, 1, 4, 1, 3, 0, 2, 0, 3, 1, 2, 1, 0, 0, 0, 0.9999998807907104, 1, 0, 1,
+    0.9999998807907104 ]);
 var indices = new Uint32Array([ 0, 1, 2, 3, 2, 1, 4, 5, 6, 7, 6, 5, 8, 9, 10, 11, 10, 9, 12, 13, 14, 15, 14, 13, 16, 17,
     18, 19, 18, 17, 20, 21, 22, 23, 22, 21 ]);
 var primitiveType = 4;
```

Comment:
I should have brought this up before, but instead can you extract these values from the gltf like in line 71-92 of `cesiumGeometryToGltfPrimitiveSpec`, just so if the test model changes, the test will still pass.